UI/window-basic-stats.cpp

```diff
@@ -429,12 +446,19 @@ void OBSBasicStats::Update()
 
 	/* ------------------------------------------- */
 	/* recording/streaming stats                   */
+	int recordingIndex = (int)outputLabels.size() - 1;
+	int numOutputs = (int)streamingOutputs.size();
 
-	outputLabels[0].Update(strOutput, false);
-	outputLabels[1].Update(recOutput, true);
+	for (int i = 0; i < recordingIndex; i++) {
+		if (i < numOutputs) {
+			outputLabels[i].Update(streamingOutputs[i], false);
+		}
```

Comment:
```suggestion
		if (i < numOutputs)
			outputLabels[i].Update(streamingOutputs[i], false);
```

Reply:
or, maybe `i < recordingIndex && i < numOutputs` at the 2nd sentence of the `for` statement.